src/components/Navigation/Link.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <li ref="link">
+    <router-link
+      @click="$emit('getoffsetposition', $event)"
+      class="text-white text-3xl relative mr-20"
+      :to="to"
+      >{{ name }}</router-link
+    >
+  </li>
+</template>
+
+<script>
+export default {
+  emits: ["getoffsetposition"],
```

Comment:
camel case

Reply:
I o ile się nie myle jak wywołujesz w template po $emit to nie musisz tu zapisywać emitów